src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -141,4 +146,33 @@ public void MercuryCuratedNavigationTest_004_navigateThroughDifferentUrl() {
     section.isAlertNotificationVisible();
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
   }
+
+  @Test(groups = "MercuryCuratedNavigationTest_005")
+  @Execute(onWikia = MercuryWikis.MERCURY_CC)
+  public void MercuryCuratedNavigationTest_005_redirectToExistingArticle() {
+    ArticleContent content = new ArticleContent();
+    String redirect = WikiTextContent.REDIRECT + WikiTextContent.LINK_PREFIX +
+                      MercurySubpages.CC_REDIRECT_DESTINATION_1 + WikiTextContent.LINK_POSTFIX;
+    content.push(redirect, MercurySubpages.CC_REDIRECT_SOURCE_1);
+
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openCuratedMainPage(wikiURL, MercurySubpages.CC_REDIRECT_SOURCE_1);
+    Assertion.assertEqualsIgnoreCase(article.getArticleTitle(),
+                                     MercurySubpages.CC_REDIRECT_DESTINATION_1);
+  }
+
+  @Test(groups = "MercuryCuratedNavigationTest_006")
+  @Execute(onWikia = MercuryWikis.MERCURY_CC)
+  public void MercuryCuratedNavigationTest_006_redirectToNotExistingArticle() {
+    ArticleContent content = new ArticleContent();
+    String redirect = WikiTextContent.REDIRECT + WikiTextContent.LINK_PREFIX +
+                      MercurySubpages.CC_REDIRECT_DESTINATION_2 + WikiTextContent.LINK_POSTFIX;
+    content.push(redirect, MercurySubpages.CC_REDIRECT_SOURCE_2);
+
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openCuratedMainPage(wikiURL, MercurySubpages.CC_REDIRECT_SOURCE_2);
+    article.verifyURLcontains(MercurySubpages.CC_REDIRECT_SOURCE_2);
+    MercuryErrorComponentObject mercuryError = new MercuryErrorComponentObject(driver);
```

Comment:
consider passing to constructor message and alert type (error, warning, info) - this way each instance of this component would be strictly bound to particular alert on page